src/bin/zos-cli.js

```diff
@@ -1,8 +1,18 @@
 #! /usr/bin/env node
 
 import { Logger } from 'zos-lib'
+import { lockSync } from 'lockfile';
 import program from './program'
 
+// Acquire file lock to ensure no other instance is running
+const LOCKFILE = '.zos.lock';
+try { 
+  lockSync(LOCKFILE, { retries: 0 }) 
+} catch (e) { 
+  console.error(`Could lock file '${LOCKFILE}'. Ensure no other instance of zos is running.`); 
```

Comment:
I'd prefer something clearer like `Cannot run more than one instance of zos at the same time. If you are sure you are not running more than once, please remove .zos.lock` or sth like that

Reply:
Updated to `Cannot run more than one instance of 'zos' at the same time.\nIf you are sure that no other instances are actually running, manually remove the file ${LOCKFILE} and try again.`